docs/d/server_certificate.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: "oneview"
+page_title: "Oneview: server_certificate"
+sidebar_current: "docs-server_certificate"
+description: |-
+ Gets information about an existing server_certificate.
+---
+
+# oneview\_server\_certificate
+
+Use this data source to access the attributes of a server certificate.
+
+## Example Usage
+
+```hcl
+data "oneview_server_certificate" "test" {
+ name = "Test server certificate"
+}
+
+output "oneview_server_certificate_value" {
+ value = "${data.oneview_server_certificate.test.uri}"
+}
+```
+
+## Argument Reference
+
+* `alias_name` - (Required) The alias name of an existing server certificate.
```

Comment:
Making both attributes required is not correct. Describe correctly that any one of these attributes can be used to get the Server certificate.